src/search/index.js

```diff
@@ -184,11 +184,11 @@ export default class CoProcureSearch extends HTMLElement {
     if(this.query || this.prepop) {
       if(this.query && this.query.indexOf('"')<0) {
         url += `&q=(and `;
-        decodeURIComponent(this.query).split(' ').forEach( (term) => {
-          url += ` '${term}'`;
+        this.query.split(' ').forEach( (term) => {
```

Comment:
So just checking, this logic here was to do exact phrase searches when there were quotes on a term, and that still works?

Reply:
Excellent question. Using the string "breath testing," I observed that the string (with quotes) gets the same number of results locally and on prod *and* that there are somewhat more results if searching without quotes around the string.

TL;DR: Yep, I think it's still working properly.